tools/build-dumborc.py

```
@@ -0,0 +1,77 @@
+
+
+def build(board, translation, srcdir):
+    cmake_options = " ".join(["-D%s=%s" % (key, value) for key, value in boards[board].items()])
+    cwd = os.getcwd()
+    if not os.path.exists("output"):
+        os.mkdir("output")
+    path = tempfile.mkdtemp()
+    os.chdir(path)
+    command = "cmake %s -DTRANSLATIONS=%s -DDUMBORC_RELEASE=YES %s" % (cmake_options, translation, srcdir)
+    print(command)
+    os.system(command)
```

Ruff (0.15.15)

[error] Starting a process with a shell, possible injection detected

(S605)